scss/_tokens.scss

```diff
@@ -0,0 +1,12 @@
+// Opacity
```

Comment:
Will this file be directly extracted from the design token management tools in the future ?
If not I would maybe call it core-tokens and set system/semantic tokens in another token file than variables.

Reply:
> Will this file be directly extracted from the design token management tools in the future ?

Yes it will, but we need to define if we split this file or not.

> If not I would maybe call it core-tokens

The name can be changed, no problem. I just created a fake file for now, waiting for a cool name for it :)

> and set system/semantic tokens in another token file than variables

Not entirely sure whether we could do that. We probably could, but it will also depend on how these system/semantic tokens will depend on each other.

Reply:
yeah, in fact I was thinking about b-brand and countries and how to make it simpler to know where I can change values. core-tokens should not be changed, and for variables, well it will be a mix of a lot of things, not mensionning utilities. It's still a little obscure for me on some point on how we will map values from the dictionnary to the format attented by Bootstrap. I was thinking at first as a sort of mapping cascade but it's hard to keep genericity.

Reply:
Yeah, totally agree with your point to clarify usage, but I'm not yet sure on how to find the right balance between files, especially at the semantic level, versus/with the `_variables.scss`. 

I think that in v6, Bootstrap should split this file, but since it's not the case yet, we need to find the best approach for basic users, and B-Brands.

Let's define together what's our starting point for these first PRs (to start with something), and let's challenge ourselves regularly, and this system, to enhance it along the way.
We'll be safe while being still in v0.x versions, so we'll be able to experiment 🔬

Reply:
Based on this discussion and internal ones with the rest of the team, I tried out something with https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2573/commits/1fb9a33a3e45dd591a3d308bc7141dc0b6df4f87